prusti-tests/tests/verify_overflow/pass/competitve_programming/dp/TheJackpot_10684.rs

```diff
@@ -0,0 +1,120 @@
+//https://onlinejudge.org/index.php?option=com_onlinejudge&Itemid=8&category=24&page=show_problem&problem=1625
```

Comment:
It would be useful to mention which functions are “real” code and which are there just for verification purposes.

Reply:
I rearranged the code. Now,  the mail solution function is under "Solution" comment

Reply:
Thanks!